books/RayTracingTheNextWeek.html

```diff
@@ -39,28 +39,27 @@
 
 Motion Blur
 ====================================================================================================
-When you decided to ray trace, you decided that visual quality was worth more than run-time. In your
-fuzzy reflection and defocus blur you needed multiple samples per pixel. Once you have taken a step
-down that road, the good news is that almost all effects can be brute-forced. Motion blur is
-certainly one of those. In a real camera, the shutter opens and stays open for a time interval, and
-the camera and objects may move during that time. Its really an average of what the camera sees over
-that interval that we want.
+When you decided to ray trace, you decided that visual quality was worth more than run-time. When
+rendering fuzzy reflection and defocus blur, we used multiple samples per pixel. Once you have taken
+a step down that road, the good news is that almost _all_ effects can be similarly brute-forced.
+Motion blur is certainly one of those.
+
+In a real camera, the shutter remains open for a short time interval, during which the camera and
+objects in the world may move. To accurately reproduce such a camera shot, we seek an average of
+all the instant images that the camera perceives while its shutter is open to the world.
 
 
 Introduction of SpaceTime Ray Tracing
 --------------------------------------
-We can get a random estimate by sending each ray at some random time when the shutter is open. As
-long as the objects are where they should be at that time, we can get the right average answer with
-a ray that is at exactly a single time. This is fundamentally why random ray tracing tends to be
-simple.
-
-The basic idea is to generate rays at random times while the shutter is open and intersect the model
-at that one time. The way it is usually done is to have the camera move and the objects move, but
-have each ray exist at exactly one time. This way the “engine” of the ray tracer can just make sure
-the objects are where they need to be for the ray, and the intersection guts don’t change much.
+We can get a random estimate of a single photon by sending a single ray at some random instant in
```

Comment:
Since we're using sRGB (super duper simplification *grumble* *grumble*), any one of our rays is actually a linear mixture of photons. We cannot, as the text outlines, get a "random estimate" of a single photon.

Wrong, but definitely a *good* change.

Reply:
Thinking of a single ray as a single photon is --in my opinion-- an immensely fruitful mental model, so I recommend that forever. (See Above)

Reply:
“... a random estimate of a single (simplified) photon by sending ...”?

Reply:
I don't like presenting simplified models as true to reality. Always bugged me when I learned that electrons don't _really_ "orbit the nucleus like planets", and wished that the more complex reality was at least hinted at.

Reply:
A phoxel!